packages/website/blog/2024-05-11-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -0,0 +1,310 @@
+---
+authors:
+  - image_url: /img/team/joshuakgoldberg.jpg
+    name: Josh Goldberg
+    title: typescript-eslint Maintainer
+    url: https://github.com/JoshuaKGoldberg
+description: Announcing the release of typescript-eslint's v8 beta, including its changes and timeline.
+slug: announcing-typescript-eslint-v8-beta
+tags: [breaking changes, typescript-eslint, v7, v8]
+title: Announcing typescript-eslint v8 Beta
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[typescript-eslint](https://typescript-eslint.io) is the tooling that enables standard JavaScript tools such as [ESLint](https://eslint.org) and [Prettier](https://prettier.io) to support TypeScript code.
+We've been working on a set of breaking changes and general features that we're excited to get in front of users soon.
+And now, we're excited to say that typescript-eslint v8 is ready for public beta testing! 🎉
+
+Our plan for typescript-eslint v8 is to:
+
+1. Have users try out betas starting in May of 2024
+2. Respond to user feedback for the next ~1-2 months
+3. Release a stable version summer of 2024
+
+Nothing mentioned in this blog post is set in stone.
+If you feel passionately about any of the choices we've made here —positively or negatively— then do let us know on [the typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg)'s `#v8` channel!
```

Comment:
I'm still confused... Don't we want the spaces here? lmao 😆 
```suggestion
If you feel passionately about any of the choices we've made here — positively or negatively — then do let us know on [the typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg)'s `#v8` channel!
```
Especially granted https://github.com/typescript-eslint/typescript-eslint/pull/9022#discussion_r1594543484 ?? I think we're both pro-spaces unless I misunderstand

Otherwise I approve of all the m-dash changes 😄 👍

Reply:
lol the way it is is my current style... but I am a stubborn anti-establishment person. accepting suggestion 😂

Reply:
ok lol i just misunderstood your comment then; I'm not trying to bulldoze your personal touch! ❤️  Just couldn't tell what was intentionally changed/left.